tests/_runtime/packages/test_pypi_package_manager.py

```diff
@@ -64,7 +67,14 @@ async def test_uninstall(mock_run: MagicMock):
     result = await manager.uninstall("package1 package2")
 
     mock_run.assert_called_once_with(
-        ["pip", "uninstall", "-y", "package1", "package2"],
+        [
+            "pip",
+            f"--python {PY_EXE}",
```

Comment:
i dont think this is correct, these should be separate entries in the list

Reply:
ooooh